apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/IterableContainsInOrderOnlyElementsOfAssertionSpec.kt

```diff
@@ -41,6 +59,29 @@ class IterableContainsInOrderOnlyElementsOfAssertionSpec : Spek({
             a: Double?,
             aX: Array<out Double?>
         ): Expect<Iterable<Double?>> = expect contains o inGiven order and only elementsOf listOf(a, *aX)
+
+        private val containsExactlyElementsOfShortcutFun: KFunction2<Expect<Iterable<Double>>, Iterable<Double>, Expect<Iterable<Double>>> =
+            Expect<Iterable<Double>>::containsExactlyElementsOf
+
+        private fun getContainsShortcutPair() = containsExactlyElementsOfShortcutFun.name to Companion::containsExactlyElementsOfShortcut
+
+        private fun containsExactlyElementsOfShortcut(
+            expect: Expect<Iterable<Double>>,
+            a: Double,
+            aX: Array<out Double>
+        ): Expect<Iterable<Double>> = expect containsExactlyElementsOf(a glue aX)
+
+        private val containsExactlyElementsOfNullableShortcutFun: KFunction2<Expect<Iterable<Double?>>, Iterable<Double?>, Expect<Iterable<Double?>>> =
+            Expect<Iterable<Double?>>::containsExactlyElementsOf
+
+        private fun getContainsNullableShortcutPair() = containsExactlyElementsOfNullableShortcutFun.name to Companion::containsExactlyElementsOfNullableShortcut;
+
+        private fun containsExactlyElementsOfNullableShortcut(
+            expect: Expect<Iterable<Double?>>,
+            a: Double?,
+            aX: Array<out Double?>
+        ): Expect<Iterable<Double?>> = expect containsExactlyElementsOf(a glue aX)
```

Comment:
please change to (also for IterableContainsInAnyOrderAtLeast1ElementsOfAssertionsSpec, I missed it in the review)
```suggestion
        ): Expect<Iterable<Double?>> = expect containsExactlyElementsOf listOf(a, *aX)
```

Reply:
ok then i'll do it in same commit if it's not problem

Reply:
Can you please fix it also for IterableContainsInAnyOrderAtLeast1ElementsOfAssertionsSpec

Reply:
i did it in other commit as in this commit i did not have that part of the code

Reply:
good as well